src/components/inspectors/TimerExpression.vue

```diff
@@ -61,6 +61,7 @@
             :class="{'date-disabled' : ends !== 'ondate'}"
             :disabled="ends !== 'ondate'"
             :placeholder="$t('End date')"
+            data-format="datetime"
```

Comment:
When the date-picker only deals with date formats, it sets the time portion to the start of the day, but the timer expression component sets the time portion to the same time as the start date. This seems to create a loop.

There's no such time resettnig when the date format is set to "datetime".